src/CommandLineUtils/Internal/CommandLineProcessor.cs

```diff
@@ -31,12 +30,10 @@ private CommandLineApplication _currentCommand
 
         public CommandLineProcessor(
             CommandLineApplication command,
-            ParserConfig config,
```

Comment:
With this change, is there any point to keeping the ParserConfig class?

It seems this is sort of going the opposite direction I was hoping to go in #298. I was hoping to extract the parsing behavior of CommandLineApplication into a separate class completely.

Reply:
If we want to make the parser config per command, we have to associate them. If we want to inherit the config from parent command, we can’t do it inside ParserConfig as it has no info of its parent.

Reply:
> make the parser config per command

That's probably a bigger refactor than we need right now. 

With this PR as is right now, does the `ParserConfig` class do anything? Or can it be deleted?

Reply:
The `ParserConfig` is still needed. It is a property of  `CommandLineApplication`. It tells the `CommandLineApplication` that each config is set or not.

Reply:
Sure we can move the two properties of `ParserConfig` to `CommandLineApplication` as private fields and then delete `ParserConfig`.

Reply:
I understand that you want to avoid "big class". However, the `UnrecognizedArgumentHandling` and `OptionNameValueSeparators` are already published API about `CommandLineApplication`. It will be a breaking change if we remove it from `CommandLineApplication`.
As I said in https://github.com/natemcmaster/CommandLineUtils/issues/370#issuecomment-650574378, this PR has no open API change.